blueprints/ember-cli-mocha/index.js

```diff
@@ -12,17 +12,14 @@ module.exports = {
   afterInstall: function() {
     var addonContext = this;
 
-    return this.addBowerPackageToProject('ember-mocha', '~0.8.0')
-      .then(function() {
-        return addonContext.addBowerPackageToProject('ember-cli/ember-cli-test-loader', '0.1.3');
-      })
-      .then(function() {
-        return addonContext.addBowerPackageToProject('ember-cli/ember-cli-shims', '0.0.3');
-      })
-      .then(function() {
-        if ('removePackageFromProject' in addonContext) {
-          return addonContext.removePackageFromProject('ember-cli-qunit');
-        }
-      });
+    return this.addBowerPackagesToProject([
```

Comment:
This was broken by [this changeset in ember-cli](https://github.com/ember-cli/ember-cli/commit/f87760ca966dfa1653e54a7c1349fd658c94c121).